### ISSUE TITLE
Intellisense with snippets

### ISSUE THREAD

**Issue by hpetriffer** [X-Sharp/XSharpPublic#723]
In C# if I write e.g. "prop" , I get allways also the snippets as result for Intellisense. 
In X# I have to use always CTRL + K + X to select a snippet. Is it possible to get the snippet also as a suggestion. First it can be triggered also with CTRL+.

Thanks

**Comment by RobertvanderHulst** [comment]
Good idea. We are going to change the code completion for the next build to use the asynchronous API. We will also offer the option to select which kind of options will be included in the completion list. One of these options will be the snippets.
